### PR TITLE
Ultra Moon Hawlucha Any% - adding to base repository

### PR DESCRIPTION
**Pull request by Caznovia**
**Please include the route in the name of the PR (for example, `AS Any% - Fix Archie 2`)**

## Route ID (for example, `alpha-sapphire-any-percent`)

ultra-moon-hawlucha

## Summary of changes

adding USUM Hawlucha Any%

## Checklist
* [x] I have tested these changes and Ranger does not display any errors or warnings.
* [x] All image embeds have relative paths.
* [x] The route contains all fights required for the category.

### CODE REVIEW

File: ultra-moon-hawlucha/route.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Ultra Moon Hawlucha Any%",
```
Comment (Corvimae) [inline ultra-moon-hawlucha/route.json]: Please don't include the name of the game in the title - Ranger will show it already via the `game` property.

File: ultra-moon-hawlucha/route.mdr [hunk 2]
```diff
@@ -0,0 +1,2120 @@
...
+
+:::::trainer[Kiawe]
+  :::pokemon[Arcanine]{info="has Extreme Speed" infoColor=gray}
+    - _If you have 2 X Defenses:_
+      - X Defense
+      - X Attack (x2 if you want to avoid a range)
+      - Acrobatics
+    - _Otherwise:_
+      - X Attack
+      - Acrobatics
```
Comment (Corvimae) [inline ultra-moon-hawlucha/route.mdr:1726]: Is the route successfully rendering when you drag it manually? nested lists like this often have issues due to a bug w/ the processor

Comment (Caznovia) [reply ultra-moon-hawlucha/route.mdr:1726]: Looks fine to me
Screenshot_128

File: ultra-moon-hawlucha/route.json (hunk 1 above)
Comment (Caznovia) [reply ultra-moon-hawlucha/route.json]: fixed.

File: ultra-moon-hawlucha/route.mdr (hunk 2 above)
Comment (Corvimae) [reply ultra-moon-hawlucha/route.mdr:1726]: cool, was just making sure.

**Pull request merged**